packages/app/features/dashboard/beatmaker/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useUserCollectionLazyQuery } from "app/api/graphql";
+import { isSome } from "app/utils";
+import { Text, View } from "app/design-system";
+import ProfileBeatsList from "app/features/dashboard/profile/profileBeatsList";
+import { useBeatmakerParam } from "app/hooks/param/useBeatmakerParam";
+import { useEffect } from "react";
+import { useRouter } from "solito/router";
+import { UserAvatar } from "app/ui/userAvatar";
+
+export default function BeatmakerScreen() {
+  const params = useBeatmakerParam();
+  const [query, { data, loading }] = useUserCollectionLazyQuery();
+  const { back } = useRouter();
+  const entries = data?.entries?.filter(isSome) ?? [];
+
+  useEffect(() => {
+    if (params?.id) {
+      query({
+        variables: {
+          userId: params.id,
+        },
+      });
+    }
+  }, [params, query]);
+
+  if (!params) {
+    back();
+    return null;
+  }
```

Comment:
will this ever gonna happen?

Reply:
Maybe for one render, but because I use params below I (and typescript) wana be sure that it is not null.

Reply:
we definitely don't want to go back in that case;